pkg/controller/perconaservermongodb/version_test.go

```diff
@@ -307,3 +328,528 @@ func Test_majorUpgradeRequested(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionMeta(t *testing.T) {
+	q, err := resource.ParseQuantity("1Gi")
+	if err != nil {
+		t.Fatal(err)
+	}
+	tests := []struct {
+		name        string
+		cr          api.PerconaServerMongoDB
+		want        VersionMeta
+		clusterWide bool
+		helmDeploy  bool
+	}{
+		{
+			name: "TestMinimalCR",
```

Comment:
With these test names, you can use spaces to name them, this can `Test minimal CR`, the next one can be `Test full CR with old version`, and so on...

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1139/commits/9d0afcac8604583fc066a8829644e63ec3820b12